distribution/docs/src/main/resources/_contents/_plugins/historian-policy-plugin-contents.adoc

```diff
@@ -0,0 +1,19 @@
+
+==== Historian Plugin
```

Comment:
So technically there is no longer a Historian "plugin".  Well there is and isn't.

Now, the Historian is embedded in the catalog framework and is configurable via a toggle in the catalog admin config. There is also still a Historian Policy plugin that is always enabled, which allows only users with the `system-history` role to create/modify/delete history items.